backend/endpoints/rom.py

```diff
@@ -89,11 +89,22 @@ def upload_roms(request: Request, p_slug: str, roms: list[UploadFile] = File(...
     log.info(f"Uploading files to: {platform_fs_slug}")
     if roms is not None:
         roms_path = build_upload_roms_path(platform_fs_slug)
+        for rom in roms: #TODO: Refactor code to avoid double loop
```

Comment:
This is fine and consistent for now, but you're right we should refactor it at some point to fail gracefully.